classes/Loggly.cls

```diff
@@ -462,32 +462,50 @@ global with sharing class Loggly {
 	*
 	* @param logs A set of log messages (in JSON format) to send to endpoint
 	*/
-	@future (callout=true)
 	global static void sendLogs(Set<String> logs) {
-		List<String> logList = new List<String>();
-		logList.addAll(logs);
+		try {
+			if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs()) {
+				System.enqueueJob(new QueueableLogSender(logs));
+			} else {
```

Comment:
I'm not a fan of having debug message like this in code (especially in a managed package where it can't be seen).  I'd prefer to have lines 469-471 removed